.pre-commit-config.yaml

```diff
@@ -3,4 +3,10 @@ repos:
     rev: stable
     hooks:
     - id: black
-      language_version: python3
+      language_version: python3.8
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v3.4.0
+  hooks:
+    - id: check-merge-conflict
+    - id: debug-statements
+    - id: detect-private-key
```

Comment:
These looks nice. Is this something we should add to the `dear-petition` repo later?